assets/src/components/ShareButtons/ShareButtons.js

```diff
@@ -65,3 +68,12 @@ const twitterUrl = (link, title, description, account) => {
     + (description ? ` - ${encodeURIComponent(description)}` : '')
     + (account ? ` via @${encodeURIComponent(account)}&related=${encodeURIComponent(account)}` : '');
 }
+
+const utm = (utm_source, utm_medium, utm_content, utm_campaign) => {
+  return [
+    utm_source ? `utm_source=${encodeURIComponent(utm_source)}` : null,
+    utm_medium ? `utm_medium=${encodeURIComponent(utm_medium)}` : null,
+    utm_content ? `utm_content=${encodeURIComponent(utm_content)}` : null,
+    utm_campaign ? `utm_campaign=${encodeURIComponent(utm_campaign)}` : null,
+  ].filter(x => x).join('&');
```

Comment:
What's the idea of filtering the `array`? Because I don't see any specific behavior doing that.

Reply:
it filters null values, so that I don't have a `&` all by itself
```js
>> ['foo', null, 'bar'].filter(x => x)
Array [ "foo", "bar" ]
```